PWGHF/DataModel/CandidateReconstructionTables.h

```diff
@@ -553,6 +555,18 @@ auto invMassGammaToEE(const T& candidate)
   return candidate.m(array{RecoDecay::getMassPDG(kElectron), RecoDecay::getMassPDG(kElectron)});
 }
 
+template <typename T>
+auto ctV0K0Short(const T& candidate)
```

Comment:
```suggestion
auto ctV0K0s(const T& candidate)
```
probably better for consistency with `LcToK0sP`.

Reply:
The way it is now it is more consistent with https://github.com/AliceO2Group/O2Physics/blob/207d4626f94d0cedb0c00ec5d4f94a71d3feceaa/PWGHF/D2H/Tasks/taskLcToK0sP.cxx#L93 , then I should change also that. But this I have named like that to be consistent with the v0data table https://github.com/AliceO2Group/O2Physics/blob/207d4626f94d0cedb0c00ec5d4f94a71d3feceaa/PWGLF/DataModel/LFStrangenessTables.h#L126-L127
As you prefer, I don't have a strong opinion on that.

Reply:
For direct references to LF tables, columns and functions, I am fine with keeping `K0Short`. This case is a new HF code function which has no link to any LF code.